index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css" integrity="sha512-rSWTr6dChYCbhpHaT1hg2tf4re2jUxBWTuZbujxKg96+T87KQJriMzBzW5aqcb8jmzBhhNSx4XYGA6/Y+ok1vQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link title="Cocogoose" href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <!--Main Wrapper-->
+     <div class="wrapper">
+      <header>
+        <div class="nav-bar1">
+          
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="https://canon20paul.github.io/My-Portfolio/">My Page</a>
+            <a href="#">Logout</a>
+          </div>
+        
+        <nav class="nav-bar">
+
+          
+       
+          <div class="nav-item logo">
+            
+            <a href="#"><img class="logo-img" src="images/proLogo.png" alt="logo"/> 
+            <h3>REMOTE WORKERS  SUMMIT<br>2022</h3></a>
```

Comment:
- [ ] Kindly fix the position of this element because half of it is hidden in the top 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190240412-97bf9069-8263-436c-a561-33a3bec60ff1.png)